processor/src/main/java/org/mapstruct/ap/model/IterableMappingMethod.java

```diff
@@ -111,7 +111,10 @@ public IterableMappingMethod build() {
             );
 
             if ( assignment == null ) {
-                ctx.getMessager().printMessage( method.getExecutable(), Message.ITERABLEMAPPING_MAPPING_NOT_FOUND );
+                ctx.getMessager().printMessage( method.getExecutable(),
```

Comment:
Do you think we really need to add the method name? The error marker will be raised right at the concerned method, so I don't think that's necessary.

I'd just say "No implementation can be generated for this method. Found no method nor implicit conversion for...".